src/main/java/banking/database/Operations.java

```diff
@@ -0,0 +1,149 @@
+package banking.database;
+
+import banking.processing.Account;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class Operations {
+
+    private final String url;
+    private final String INSERT_INTO = "INSERT INTO card (number, pin) VALUES (?, ?)";
+    private final String SELECT_CARDNUMBER = "SELECT count(number) FROM card WHERE number = (?)";
+    private final String SELECT_ALL = "Select * FROM card";
```

Comment:
Can you use the same name convention - uppercase?

Reply:
fixed